ledger/simulation/trace.go

```diff
@@ -64,6 +64,11 @@ func makeTxnGroupResult(txgroup []transactions.SignedTxn) TxnGroupResult {
 	return groupResult
 }
 
+// SimulatorConfig packs simulation related configurations, that overlaps the configurations in real transactions.
+type SimulatorConfig struct {
+	UnLimitLog bool
+}
+
```

Comment:
why not in simulator.go?

Reply:
sure, that makes more sense. see https://github.com/algorand/go-algorand/pull/5247/commits/21a80dbc2e302657e68d7404881056eb101c733b